.github/ISSUE_TEMPLATE/2_bug-or-performance-issue-report.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Bug or performance issue report
+about: Create a report to help us improve oneDPL
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug or performance issue is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+
+The following information might be useful: 
```

Comment:
I think some additional useful information to request from the user would be their OS, the device the issue was produced with, and GPU driver version for issues that occur on GPUs.

Reply:
Do you think that this is something we need from the start or do you think that most issues can be reproduced without needing this?  
For me it seems like these are good first follow up questions if we are unable to reproduce without it.

What do others think?

Reply:
I think it is very dependent on the type of issue, so to me it would also be okay to just ask these questions as follow-up if we think it may be too much.

Alternatively, we could have a more open-ended optional request. Something like: "[Optional] Any system configuration or version information that may be needed to reproduce the issue:".

Reply:
I agree some optional blanket statement is worthwhile and we kind of have this with `**Additional context**`, but perhaps we can give some examples of things that could be useful in that description in a concise list form.

Reply:
Ah, I missed that. I would be in favor then of adding some examples there instead of anything here.

Reply:
I added lines with OS, device and drivers version